modules/administration-guide/examples/snip_che-getting-the-list-of-relevant-images.adoc

```diff
@@ -11,7 +11,8 @@ $ curl -sSLo- https://raw.githubusercontent.com/eclipse/che-operator/master/olm/
 ====
 [subs="+attributes,+quotes,+macros"]
 ----
-$ curl -sSLo- https://raw.githubusercontent.com/eclipse/che-operator/master/olm/eclipse-che-preview-openshift/deploy/olm-catalog/eclipse-che-preview-openshift/{prod-ver-patch}/eclipse-che-preview-openshift.v{prod-ver-patch}.clusterserviceversion.yaml | \
-  yq -r '.spec.relatedImages[]'
+$ curl -sSLo- https://raw.githubusercontent.com/eclipse-che/che-operator/main/deploy/olm-catalog/stable/eclipse-che-preview-openshift/manifests/che-operator.clusterserviceversion.yaml | \
```

Comment:
so, the version is not going to be specified anymore?

Reply:
That's exactly the question I was about to investigate.